src/runtime/styled.js

```diff
@@ -124,6 +132,7 @@ function jsx(type, props, ...children) {
     const { css, ...childProps } = props;
     childProps.style = varsToStyles(childProps, css[1]);
     childProps.className = propsToStyles(childProps, css[0] || css, true);
+    childProps.className += resolveVariants(css[2]);
```

Comment:
do you need another space to separate these?

Reply:
not with the reduce since the first one is `''`